x-pack/plugins/apm/server/routes/traces.js

```diff
@@ -30,7 +30,7 @@ export function initTracesApi(server) {
     handler: (req, reply) => {
       const { traceId } = req.params;
       const { setup } = req.pre;
-      return getTrace({ traceId, setup })
+      return getTrace(traceId, setup)
```

Comment:
Did you just change this because destructured types are uglier than inline args? 😆

Reply:
Yes! :D You okay with that?